python/vespa/vespa/application.py

```diff
@@ -87,7 +90,7 @@ def query(
         if debug_request:
             return VespaResult(vespa_result={}, request_body=body)
         else:
-            r = post(self.search_end_point, json=body)
+            r = post(self.search_end_point, json=body, cert=self.cert)
```

Comment:
Not sure if this works against the local docker instance.

Reply:
Yes, it works.